src/Pages/Nodes.razor

```diff
@@ -237,7 +262,19 @@
         {
             arg.Item.UpdateDatetime = DateTimeOffset.Now;
             arg.Item.CreationDatetime = DateTimeOffset.Now;
-            var addResult = await NodeRepository.AddAsync(arg.Item);
+            (bool, string?) addResult;
+            var restoredNode = await NodeRepository.GetByPubkey(arg.Item.PubKey);
+            if (restoredNode != null)
+            {
+                restoredNode.EnabledNode = true;
+                restoredNode.Endpoint = arg.Item.Endpoint;
+                restoredNode.ChannelAdminMacaroon = arg.Item.ChannelAdminMacaroon;
+                addResult=  NodeRepository.Update(restoredNode);
```

Comment:
What if the user changed the other properties like name and description, etc?
You should update all fields of restoredNode